2-mandatory/6-budgets.js

```diff
@@ -11,14 +11,21 @@ Should give return the answer of 62600.
 **/
 
 function getBudgets(peopleArray) {
+const getBudgets = peopleArray.map(elements => elements.budget) // 1 access only budgets and turned it to an array
+return getBudgets.reduce((acc, curr) => { // 2 then use reduce method to ge the sume of the array
+   acc += curr
```

Comment:
I would have avoided shortening these var names, but tests pass and code is good otherwise